Changelog for new blueprints (#1561)
FlowFuse/website: src/changelog/2024/01/new-blueprints.md

---
title: New Blueprints added
date: 2024-01-22 13:00:00.0
authors: ["marian-demme"]
tags:
    - changelog
---

> We added new blueprints to FlowFuse Cloud. They are available on FlowFuse Cloud. Additionally, upon request, all our Teams and Enterprise Self-Hosted customers gain access to this collection.

Reviewer: ```suggestion
We have added some new blueprints to FlowFuse Cloud to help users get started with a range of common manufacturing scenarios.

They are available to all users on FlowFuse Cloud. Additionally, upon request, all our Teams and Enterprise Self-Hosted customers gain access to this collection.
```

List of new Blueprints:

> - Andon Live

Reviewer 2: ```suggestion
- [Andon Live](/blueprints/manufacturing/andon-live/)
```

> - CRUD for Mongo DB

Reviewer 2: ```suggestion
- [CRUD for Mongo DB](/blueprints/getting-started/crud/)
```

> - 5S Manufacturing checklist

Reviewer 2: ```suggestion
- [5S Manufacturing checklist](/blueprints/manufacturing/5s-checklist/)
```

> - Performance Operator Terminal

Reviewer 2: ```suggestion
- [Performance Operator Terminal](/blueprints/manufacturing/performance-overview/)
```

---
title: New Blueprints added
date: 2024-01-22 13:00:00.0
authors: ["marian-demme"]
tags:
    - changelog
---
We have added some new blueprints to FlowFuse Cloud to help users get started with a range of common manufacturing scenarios.

They are available to all users on FlowFuse Cloud. Additionally, upon request, all our Teams and Enterprise Self-Hosted customers gain access to this collection.

List of new Blueprints:
- [Andon Live](/blueprints/manufacturing/andon-live/)
- [CRUD for Mongo DB](/blueprints/getting-started/crud/)
- [5S Manufacturing checklist](/blueprints/manufacturing/5s-checklist/)
- [Performance Operator Terminal](/blueprints/manufacturing/performance-overview/)

> Curious about what Blueprints are? [Click here](https://flowfuse.com/blog/2023/10/blueprints/)
> For an overview of all available Blueprints, [click here](https://flowfuse.com/blueprints/)

Reviewer: ```suggestion
Curious about what Blueprints are? Read our recent blog post that [introduced Blueprints to the FlowFuse](https://flowfuse.com/blog/2023/10/blueprints/).

For an overview of all available Blueprints, check out our [Blueprint Library](https://flowfuse.com/blueprints/).
```

Making the link text more google friendly
